shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/datasource/pool/metadata/DataSourcePoolMetaData.java

```diff
@@ -50,6 +50,13 @@ public interface DataSourcePoolMetaData extends TypedSPI {
      */
     Map<String, String> getPropertySynonyms();
     
+    /**
+     * Get jdbc url synonym field names.
+     *
+     * @return jdbc url synonym field names
+     */
+    Collection<String> getJdbcUrlSynonymFieldNames();
```

Comment:
Why name it `JdbcUrlSynonymFieldNames`? It seems that `registerMbeans` has nothing to do with JDBC URL.

Reply:
`JdbcUrlPropertiesFieldName`  is from `DataSourcePoolFieldMetaData.getJdbcUrlPropertiesFieldName`  for interface

Reply:
```
public final class HikariDataSourcePoolFieldMetaData implements DataSourcePoolFieldMetaData {
    ...
    ...
    @Override
    public String getJdbcUrlPropertiesFieldName() {
        return "dataSourceProperties";
    }
}
```

Reply:
replace  `JdbcUrlProperties`  with `customPoolProperties`  is better I think